src/blr/Snakefile

```diff
@@ -123,15 +131,13 @@ rule mark_duplicates:
         bam = "mapped.sorted.tag.mkdup.bam"
     input:
         bam = "mapped.sorted.tag.bam"
-    log:
-        metrics = "picard_mkdup_metrics.log",
-        stderr = "picard_mkdup.log"
+    log: "mark_duplicates.log"
+    threads: 20
     shell:
-        "picard -Xms{config[heap_space]}g MarkDuplicates"
-        " I={input.bam}"
-        " O={output.bam}"
-        " M={log.metrics}"
-        " ASSUME_SORT_ORDER=coordinate 2> {log.stderr}"
+        "sambamba markdup "
```

Comment:
Please remove the space before the last `"` in this and the next line.

Reply:
Fixed!